docs/components/modeler/what-is-camunda-modeler.md

```diff
@@ -1,11 +1,11 @@
 ---
-id: overview
-title: Overview
+id: what-is-camunda-modeler
+title: What is Camunda Modeler?
 ---
 
 Any executable process needs a **BPMN diagram** designed and configured beforehand.
 
-The design is used to specify the flow of a process. Thereby, different events must be configured in the model so the workflow engine understands what is to be executed in this step.
+The design is used to specify the flow of a process. Thereby, different events must be configured in the model so the workflow engine understands what must executed.
```

Comment:
The way I explain this: 

* You draw a BPMN diagram that represents your process, structurally / visually and is easily understood by different stakeholders.

* To implement it you add specific meta-data (_implementation details_) such as which particular implementation shall be invoked once the process reaches a task.


==> The Camunda Modeler (Desktop, Cloud, any) is, along with your IDE, the camunda implementation tool

Reply:
Adjusted our wording in What is Camunda Modeler in the latest commit -- does this help better describe Modeler's capabilities, or are there additional particular details you would add?